File: code/packages/csharp/csharp-parser/tests/CodingAdventures.CSharpParser.Tests/CSharpParserTests.cs
namespace CodingAdventures.CSharpParser.Tests;

public sealed class CSharpParserTests
{
    [Fact]
    public void PingReturnsPackageName()
    {
        Assert.Equal("csharp-parser", new CodingAdventures.CSharpParser.CSharpParser().Ping());
    }
}

File: code/packages/fsharp/csharp-parser/tests/CodingAdventures.CSharpParser.Tests/CSharpParserTests.fs
namespace CodingAdventures.CSharpParser.Tests

open CodingAdventures.CSharpParser.FSharp
open Xunit

module CSharpParserTests =
    [<Fact>]
    let pingReturnsPackageName () =
        Assert.Equal("csharp-parser", CSharpParser().Ping())